PJ05/src/Exceptions/InvalidUsernameException.java

```diff
@@ -0,0 +1,16 @@
+package Exceptions;
+
+import Field.User;
+
+public class InvalidUsernameException extends Exception{
```

Comment:
Remember to run "reformat code" before any commit/push.
```suggestion
public class InvalidUsernameException extends Exception {
```